src/main/kotlin/eu/kevin/api/models/general/bank/BankResponse.kt

```diff
@@ -0,0 +1,16 @@
+package eu.kevin.api.models.general.bank
+
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class BankResponse(
+    val id: String,
+    val name: String,
+    val officialName: String? = null,
```

Comment:
I just wonder whether this and `imageUri` are ever null. Because in documentation there is no indication that these fields are null. Unless I am missing something.

Reply:
There are a few banks which have this as null in prod and non-prod environments although it is not mentioned in the documentation

Reply:
Yeah, I remember same issue when we were trying to impl Android SDK. Already reported to Mike about documentation improvement.